[BUG] java.lang.RuntimeException: Failed to create driver

**Context:**
- Playwright Version: 1.17.2
- Operating System: [e.g. Windows]
- Browser: [Chrome]
- Extra: standalone Selenium Grid 4 is set up in a windows remote machine.
Getting below error while running testcase through standalone selenium grid 4(created in a remote windows machine)
playwright version used 1.17.2
```
[INFO] Running org.example.AppTest
java.lang.RuntimeException: Failed to create driver
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:54)
	at com.microsoft.playwright.impl.PlaywrightImpl.create(PlaywrightImpl.java:39)
	at com.microsoft.playwright.Playwright.create(Playwright.java:92)
```
code snippet
---------------
```java
public void shouldAnswerWithTrue()
{
    List<String> list=new ArrayList<String>();
    list.add("--start-maximized");
    Map<String, String> env = new HashMap<>();
    env.put("SELENIUM_REMOTE_URL", "http://testing.net.int:4444/");
    try(Playwright playwright = Playwright.create(new Playwright.CreateOptions().setEnv(env))){
        Browser browser = playwright.chromium().launch(new BrowserType.LaunchOptions().setHeadless(false).setSlowMo(100).setArgs(list));
        BrowserContext browserContext = browser.newContext();
        Page page = browserContext.newPage();
    }
    catch (Exception e) {
            e.printStackTrace();
        }
}
```

@yury-s As you suggested i have added DEBUG="pw:install,pw:channel" i am getting below logs.

![image](https://user-images.githubusercontent.com/32022027/164317151-555f5c89-034e-4588-aa37-9cdb3eb6a4ed.png)

C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\VFS\ProgramFilesX64\Java\jdk8\bin\java.exe -Dmaven.multiModuleProjectDirectory=C:\Users\dasatya\workspace\playwrightDemo "-Dmaven.home=C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\plugins\maven\lib\maven3" "-Dclassworlds.conf=C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\plugins\maven\lib\maven3\bin\m2.conf" "-Dmaven.ext.class.path=C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\plugins\maven\lib\maven-event-listener.jar" "-javaagent:C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\lib\idea_rt.jar=53162:C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\bin" -Dfile.encoding=UTF-8 -classpath "C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\plugins\maven\lib\maven3\boot\plexus-classworlds-2.6.0.jar;C:\ProgramData\AppV\1E69598A-0916-46CF-A75B-FBE77BCC81EA\5D0E86B0-3D1F-49EA-87B4-A2E648801851\Root\IntelliJ IDEA Community Edition 2021.1.1\plugins\maven\lib\maven3\boot\plexus-classworlds.license" org.codehaus.classworlds.Launcher -Didea.version=2021.1.1 test
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.example:playwrightDemo:jar:1.0-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: junit:junit:jar -> version 4.11 vs 4.13 @ line 33, column 17
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: junit:junit:jar -> version 4.11 vs 4.13 @ line 39, column 17
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] ---------------------< org.example:playwrightDemo >---------------------
[INFO] Building playwrightDemo 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:3.0.2:resources (default-resources) @ playwrightDemo ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\dasatya\workspace\playwrightDemo\src\main\resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ playwrightDemo ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:3.0.2:testResources (default-testResources) @ playwrightDemo ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\dasatya\workspace\playwrightDemo\src\test\resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:testCompile (default-testCompile) @ playwrightDemo ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ playwrightDemo ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.example.AppTest
2022-04-20T20:26:41.514Z pw:install created DriverJar: C:\Users\dasatya\AppData\Local\Temp\1\playwright-java-8170381812242527446
2022-04-20T20:26:41.518Z pw:install initializing driver
2022-04-20T20:26:43.592Z pw:install extracted driver from jar to C:\Users\dasatya\AppData\Local\Temp\1\playwright-java-8170381812242527446\playwright.cmd
java.lang.RuntimeException: Failed to create driver
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:61)
	at com.microsoft.playwright.impl.PlaywrightImpl.create(PlaywrightImpl.java:40)
	at com.microsoft.playwright.Playwright.create(Playwright.java:96)
	at org.example.AppTest.shouldAnswerWithTrue(AppTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.RuntimeException: Failed to install browsers, exit code: 1
	at com.microsoft.playwright.impl.DriverJar.installBrowsers(DriverJar.java:76)
	at com.microsoft.playwright.impl.DriverJar.initialize(DriverJar.java:48)
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:57)
	... 31 more
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.208 s - in org.example.AppTest
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.019 s
[INFO] Finished at: 2022-04-20T22:26:46+02:00
[INFO] ------------------------------------------------------------------------

Process finished with exit code 0

--- a/driver-bundle/src/main/java/com/microsoft/playwright/impl/DriverJar.java
+++ b/driver-bundle/src/main/java/com/microsoft/playwright/impl/DriverJar.java
@@ -26,6 +26,7 @@ import java.util.concurrent.TimeUnit;
 
 public class DriverJar extends Driver {
   private static final String PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD = "PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD";
+  private static final String SELENIUM_REMOTE_URL = "SELENIUM_REMOTE_URL";
   private final Path driverTempDir;
 
   public DriverJar() throws IOException {
@@ -55,6 +56,10 @@ public class DriverJar extends Driver {
     }
     if (skip != null && !"0".equals(skip) && !"false".equals(skip)) {
       System.out.println("Skipping browsers download because `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD` env variable is set");
+      return;
+    }
+    if (env.get(SELENIUM_REMOTE_URL) != null || System.getenv(SELENIUM_REMOTE_URL) != null) {
+      logMessage("Skipping browsers download because `SELENIUM_REMOTE_URL` env variable is set");
       return;
     }
     Path driver = driverPath();

